### PR TITLE
Add initial list of confirmed sponsors

### PR DESCRIPTION
**Pull request by dcwalk**
Hey, first attempt at this, couple thoughts:

1. Could reach out for TMAC svg
1. Can not find semaphore svg
1. Something hinky with location sponsors layout that could be adjusted
1. Nav position and shout outs on site probably could use some thinking

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+order: 2
+section: "2018"
+title: "Sponsors"
+titleDisplay: "Sponsors"
+parent: "About the Event"
+---
+
+Our Networks is a volunteer-run and would not be possible without the support of our partners and sponsors.
```
Comment (benhylau) [inline sponsors.md]: `a volunteer-run conference`

**Review by benhylau — DISMISSED** [review]

File: sponsors.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  </div>
+  <div class="four columns">
+    <a href="https://protocol.ai/" rel="noopener"><img src="/images/sponsors/protocol-labs-logo.svg" width="220" alt="Protocol Labs logo" /></a>
+  </div>
+  <div class="four columns">
+    <a href="https://dapphub.com/" rel="noopener"><img src="/images/sponsors/dapp-logo.svg" width="130" alt="Dapp.Org logo" /></a>
+  </div>
+</div>
+
+## Location Support
```
Comment (garrying) [inline sponsors.md]: Maybe `Venue Support` or `Venue Partners`?

**Review by garrying — COMMENTED** [review]
Adjusted the logo spacing and did some manual sizing to even out the sizing. I think it's okay if we didn't have SVGs for logo for now. Are we looking to include site shout outs in this PR? Maybe somewhere on the homepage might work? We could come back to it

File: sponsors.md (hunk 2 above)
Comment (dcwalk) [reply sponsors.md]: Either works!

**Comment by dcwalk** [comment]
Agree non-critical to include shoutouts in this PR, but that is something we need to think about over June

**Comment by dcwalk** [comment]
Think we are good to merge if okay with folks?

**Comment by garrying** [comment]
Good with me! ✅

**Comment by dcwalk** [comment]
Done! Y'all sometimes I think we can be more proactive on the merges. We're too polite 😂

**Pull request merged**